guides/source/action_cable_overview.md

```diff
@@ -0,0 +1,659 @@
+this would be something like: `App.cable = ActionCable.createConsumer("ws://example.com:28080")`,
+and for an in-app server, something like: `App.cable = ActionCable.createConsumer("/cable")`.
+
+The second option is to pass the server url through the `action_cable_meta_tag` in your layout.
+This uses a url or path typically set via `config.action_cable.url` 
+in the environment configuration files, or defaults to "/cable".
+
+This method is especially useful if your WebSocket url might change
+between environments. If you host your production server via https,
+you will need to use the wss scheme for your ActionCable server, but 
+development might remain http and use the ws scheme. You might use localhost
+in development and your domain in production.
+
+In any case, to vary the WebSocket url between environments, add the following
+configuration to each environment:
+
+```ruby
+config.action_cable.url = "ws://example.com:28080"
+```
+
+Then add the following line to your layout before your JavaScript tag:
+
+```erb
+<%= action_cable_meta_tag %>
+```
+
+And finally, create your consumer like so:
+
+```coffeescript
+App.cable = ActionCable.createConsumer()
+```
+
+### Other Configurations
+
+The other common option to configure is the log tags applied to the per-connection logger. Here's close to what we're using in Basecamp:
+
+```ruby
+Rails.application.config.action_cable.log_tags = [
+  -> request { request.env['bc.account_id'] || "no-account" },
+  :action_cable,
+  -> request { request.uuid }
+]
+```
+
+Your WebSocket URL might change between environments. If you host your
+production server via https, you will need to use the wss scheme
+for your ActionCable server, but development might remain http and
+use the ws scheme. You might use localhost in development and your
+domain in production. In any case, to vary the WebSocket URL between
+environments, add the following configuration to each environment:
+
+```ruby
+config.action_cable.url = "ws://example.com:28080"
+```
+
+Then add the following line to your layout before your JavaScript tag:
+
+```erb
+<%= action_cable_meta_tag %>
+```
+
+And finally, create your consumer like so:
+
+```coffeescript
+App.cable = Cable.createConsumer()
+```
+
+For a full list of all configuration options, see the `ActionCable::Server::Configuration` class.
+
+Also note that your server must provide at least the same number of
+database connections as you have workers. The default worker pool is
+set to 100, so that means you have to make at least that available.
+You can change that in `config/database.yml` through the `pool` attribute.
+
+## Running the cable server
+
+### In App
+
+Action Cable can run alongside your Rails application. For example, to
+listen for WebSocket requests on `/websocket`, mount the server at that path:
+
+```ruby
+# config/routes.rb
+Example::Application.routes.draw do
+  mount ActionCable.server => '/cable'
+end
+```
```

Comment:
Don't need this stuff any more. We by default mount at that location. We should explain how to turn this off, though, if you're going to run as standalone.